tests/foreman/performance/test_standard_prep.py

```diff
@@ -0,0 +1,169 @@
+"""Test class for Environment Preparation after a fresh installation"""
+from robottelo.common import conf, ssh
+from robottelo.cli.org import Org
+from robottelo.cli.repository import Repository
+from robottelo.cli.repository_set import RepositorySet
+from robottelo.cli.subscription import Subscription
+
+from robottelo.test import TestCase
+
+
+class StandardPrepTestCase(TestCase):
+    """Standard process of preparation after fresh install of
+    Red Hat Satellite 6:
+
+    upload manifest, change CDN address, enable repositories
+
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(StandardPrepTestCase, cls).setUpClass()
+
+        # parameters for standard process test
+        cls.savepoint = conf.properties[
+            'performance.test.savepoint1_fresh_install']
+        cls.manifest_location = conf.properties[
+            'performance.test.manifest.location']
+
+        # parameters for uploading manifests
+        cls.manifest_file = conf.properties['performance.test.manifest.file']
+        cls.org_id = conf.properties['performance.test.organization.id']
+
+        # parameters for changing cdn address
+        cls.target_url = conf.properties['performance.test.cdn.address']
+
+        # parameters for enabling repositories
+        cls.pid = conf.properties['performance.test.pid']
+
+        # [repo-id,$basearch,$releasever]
+        cls.repository_list = [
+            [168, 'x86_64', '6Server'],
+            [2456, 'x86_64', '7Server'],
+            [1952, 'x86_64', '6.6'],
+            [2455, 'x86_64', '7.1'],
+            [166, 'x86_64', '6Server'],
+            [2463, 'x86_64', '7Server'],
+            [167, 'x86_64', '6Server'],
+            [2464, 'x86_64', '7Server'],
+            [165, 'x86_64', '6Server'],
+            [2462, 'x86_64', '7Server']
+        ]
+
+    def setUp(self):
+        self.logger.info('---------------------------------')
+        self.logger.debug('Running test %s/%s',
+                          type(self).__name__, self._testMethodName)
+
+        # Restore database to clean state"""
+        self._restore_from_savepoint(self.savepoint)
+        self.logger.info('---------------------------------')
+
+    def _restore_from_savepoint(self, savepoint):
+        """Restore from a given savepoint"""
+        self.logger.info('---------------------------------')
+        self.logger.info('Reset db from /home/backup/{}'.format(savepoint))
+        self.logger.info('---------------------------------')
+        # ssh.command('./reset-db.sh /home/backup/{}'.format(savepoint))
```

Comment:
This method does nothing. It's alarming that it says "do X" but then fails to do X.